webpack.config.js

```diff
@@ -1,9 +1,22 @@
 const uid2Entrypoint = './src/uid2Sdk.ts';
 const euidEntrypoint = './src/euidSdk.ts';
-const espEntryPoint = './src/secureSignal.ts';
+const espEntryPointUid2 = './src/secureSignalUid2.ts';
+const espEntryPointEuid = './src/secureSignalEuid.ts';
+const espSharedEntryPoint = './src/secureSignal_shared.ts';
 
 const espOutput = {
-  espScript: { import: espEntryPoint, filename: 'uid2SecureSignal.js' },
+  'uid2-secureSignals': {
+    import: espEntryPointUid2,
+    filename: `uid2SecureSignal.js`,
+  },
+  'euid-secureSignals': {
+    import: espEntryPointEuid,
+    filename: `euidSecureSignal.js`,
+  },
+  'secureSignals-shared': {
```

Comment:
You don't need this third one - shared will be built into the other two scripts.

Reply:
removed